src/test/coherentsolutions.web.test/java/Managers/WebDriverManager.java

```diff
@@ -3,17 +3,20 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 
+import java.util.concurrent.TimeUnit;
+
 public class WebDriverManager {
 
     private static WebDriver driver;
 
     public static WebDriver initDriver() {
         driver = new ChromeDriver();
         driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(3, TimeUnit.SECONDS);
```

Comment:
3 seconds is very short timeout, i suggest to increase this value to 10 or even 15 sec